components/header/header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import style from './header.module.scss';
+import KedroLogo from '../../public/assets/logos/kedro';
+import DiscordLogo from '../../public/assets/logos/discord';
+import GithubLogo from '../../public/assets/logos/github';
+import BurgerIcon from '../../public/assets/icons/burger';
+import CloseIcon from '../../public/assets/icons/close';
+
+export default function Header() {
+  const [menuOpen, setMenuOpen] = useState(false);
+  return (
+    <div className={style.container}>
+      <div className={style.header}>
+        <div className={style.logo}>
+          <KedroLogo />
+          <h4 className={style.logoText}>Kedro</h4>
+        </div>
+        {/*TO DO - Update All Links*/}
+        <nav
+          className={menuOpen ? `${style.menu} ${style.active}` : style.menu}
+          onClick={() => setMenuOpen(false)}
+        >
+          <a href="#why-kedro" className={style.link}>
+            Why Kedro
+          </a>
+          <a href="#features" className={style.link}>
+            Features
+          </a>
+          <a href="#" className={style.link}>
+            Get Started
+          </a>
+          <a href="#" className={style.link}>
+            Documentation
+          </a>
+          <div className={style.iconLinks}>
+            <a className={style.link} href="https://discord.com/kedro-org">
+              <DiscordLogo />
+              <span className={style.iconText}>Discord</span>
+            </a>
+            <a className={style.githubLink} href="https://github.com/kedro-org">
```

Comment:
I believe our pattern is that these external links should open in a [new tab](https://github.com/kedro-org/kedro-website/blob/main/components/hero/hero.tsx#L19-L20).